src/config.ts

```diff
@@ -33,16 +33,16 @@ export interface Config {
 }
 
 export const config: Config = {
-    env: process.env.NODE_ENV || 'development',
+    env: /*process.env.NODE_ENV ||*/ 'development',
 
-    loginCookieName: process.env.LOGIN_COOKIE_NAME || 'photon',
-    internalToken: process.env.INTERNAL_TOKEN || 'prello123456789',
+    loginCookieName: /*process.env.LOGIN_COOKIE_NAME ||*/ 'photon',
+    internalToken: /*process.env.INTERNAL_TOKEN ||*/ 'prello123456789',
 
     server: {
-        host: process.env.SERVER_HOST || 'http://localhost',
-        port: process.env.PORT || 5000,
-        apiSuffix: process.env.API_SUFFIX || 'api',
-        authSuffix: process.env.AUTH_SUFFIX || 'auth'
+        host: /*process.env.SERVER_HOST ||*/ 'http://localhost',
```

Comment:
I guess this is because you had problem with it, but next time, try to remove the comment part ;)